package/gluon-l3roamd/files/etc/init.d/gluon-l3roamd

```diff
@@ -10,7 +10,7 @@ start_service () {
 	procd_set_param stdout 1
 	procd_set_param stderr 1
 	procd_set_param respawn ${respawn_threshold:-3660} ${respawn_timeout:-5} ${respawn_retry:-0}
-	interfaces=$(for dev in $( ubus call network.interface dump | jsonfilter -e "@.interface[@.proto='gluon_mesh' && @.up=true].device");do echo " -m $dev"; done;
+	interfaces=$(for dev in $(/usr/bin/get-mesh-ifs);do echo " -m $dev"; done;
```

Comment:
Remove the full path here, just use the name of the script.

Reply:
removed the path here but kept it in the scripts that are called by the statuspage.

Reply:
Why, do those script not have a proper PATH set?